### DISCUSSION TITLE
Moonraker v.0.9.0-0 issues?

### DISCUSSION THREAD

**Discussion by Ba22a** [Guilouz/Creality-Helper-Script-Wiki#636] [Issues]
Anyone else upgraded to Moonraker v.0.9.0-0 and found it is not talking to Klipper properly? Just had to factory reset my K1C!

Script always pulls latest version so is effectively broken at the moment.

**Comment by gosoares** [comment]
I'm having the same problem on my K1, i also did a factory reset and having the same problem after. Aparently moonraker simply does not start or crash on start.

**Comment by Tombraider2006** [comment]
I confirm. moonraker is faulty

**Comment by gosoares** [comment]
It was fixed with the last moonraker commit.
